_sass/_code-of-conduct.scss

```diff
@@ -0,0 +1,10 @@
+.code-of-conduct {
+  li {
+    padding: 2px 5px;
+
+    &:target {
+
+      background-color: #ae81ff;
```

Comment:
This color doesn’t work well, in my opinion, with other colors used in the site. I suggest `#1E5594`.

Reply:
Seems a little dark of a background, opinions?

Reply:
BTW, I arrived at the color above by checking the colors we use for blog tags against [the WebAIM Color Contrast Checker API](https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=5F9ADE) and finding that they fail every test. 😭 

So if you like my suggestion, perhaps you’d like to roll that update into this PR? If you _don’t_ like my suggestion, try playing around with the contrast checker for a color that meets accessibility best practices, and use that color for both uses.

Reply:
@StrangeWill Oh! Sorry… I meant to also say: Add `color: #ffffff;`. 😄

Reply:
Ehhh trying the lilac color we use for the writer tags. Opinions?

Reply:
I don’t like it _aesthetically,_ but [the checker likes it](https://webaim.org/resources/contrastchecker/?fcolor=232323&bcolor=A97FCE).

Reply:
Hmm, I'll ponder this a bit more, I kind of felt the same.